README.md

```diff
@@ -78,17 +78,17 @@ Host environments (e.g., the Web platform, Node.js) often provide various differ
 #### Worker instantiation
 
 ```js
-new Worker("foo.wasm", { type: "module", with: { type: "webassembly" } });
+new Worker("foo.wasm", { type: "module", if: { type: "webassembly" } });
 ```
 
 Sidebar about WebAssembly module types and the web: it's still uncertain whether importing WebAssembly modules would need to be marked specially, or would be imported just like JavaScript. Further discussion in [#19](https://github.com/littledan/proposal-module-attributes/issues/19).
 
 #### HTML
 
-The idea here would be that each import attribute, preceded by `with`, becomes an HTML attribute which could be used in script tags.
+The idea here would be that each import attribute, preceded by `if`, becomes an HTML attribute which could be used in script tags.
 
 ```html
-<script src="foo.wasm" type="module" withtype="webassembly"></script>
+<script src="foo.wasm" type="module" iftype="webassembly"></script>
```

Comment:
not that it's really up to this proposal, but `withtype`/`iftype` seem weird to me; i'd expect `if-type` or `if:type` or something else that indicated a hierarchy.

Reply:
The name here is intended to match HTML attribute naming conventions. See https://w3ctag.github.io/design-principles/#casing-rules

Reply:
that talks about concatenating multi-word things, like "max length" - "if type" isn't a single word, it's a hierarchy, and it is *very* not clear to me that "iftype" is the thing these principles dictate. it seems more like this is a case those principles simply haven't considered.

certainly this PR/repo isn't the place to discuss it tho :-) i'd appreciate a link, if you have one and when the time is relevant, to the appropriate place

Reply:
Would it be OK if the MVP web integration did not include specifying import attributes on the script tag? I think this can be delayed until we have a type on the web that motivates the use case.

Reply:
I think showing the HTML example illustrate possible uses cases, but maybe we should make it clear that it's not meant for TC39?

Reply:
Isn’t JSON the web type motivating the entire proposal?

Reply:
Yes, initially (though other module types are also considered part of the motivation), and it does not make sense to have a script tag with a JSON module--it would not do anything, since there's no way to get the exports out of a module in a script tag.

Reply:
I added a sentence to make it clear that it's outside of TC39